src/plugins/MiradorImageTools.js

```diff
@@ -116,21 +118,32 @@ class MiradorImageTools extends Component {
 
     if (!viewer || !enabled) return null;
 
+    const backgroundColor = palette.shades.main;
+    const foregroundColor = palette.getContrastText(backgroundColor);
+    const borderRight = `1px solid ${changeAlpha(foregroundColor, 0.2)}`;
+    const borderImageSource = 'linear-gradient('
+      + 'to bottom, '
+      + `${changeAlpha(foregroundColor, 0)} 20%, `
```

Comment:
Could we import and use similarly `fade`?

https://github.com/mui-org/material-ui/blob/ee6f54eb7640e3c0116e9f32c657eef3e4eeec18/packages/material-ui/src/styles/colorManipulator.js#L209-L219

This is a similar pattern that we use upstream in Mirador. https://github.com/ProjectMirador/mirador/search?q=fade&unscoped_q=fade

Reply:
Ah, brilliant, did not know about this utility :-) Will use that instead!

Reply:
Done!